src/raw.rs

```diff
@@ -454,3 +508,14 @@ impl<'de> MapAccess<'de> for BorrowedRawDeserializer<'de> {
         seed.deserialize(BorrowedStrDeserializer::new(self.raw_value.take().unwrap()))
     }
 }
+
+/// Convert a `T` into `serde_json::value::RawValue` which is a single valid
+/// JSON value.
+pub fn to_raw<T>(value: T) -> Result<Box<RawValue>, Error>
```

Comment:
I think `to_raw_value` would be a clearer name. We had decided not to call the type just `Raw`.

Could you explain a real world use case where this function would be needed?